modules/onion-services.nix

```diff
@@ -103,6 +103,14 @@ in {
     # Set sensible defaults for some services
     {
       nix-bitcoin.onionServices = {
+        spark-wallet = {
+          externalPort = 80;
+          # Enable 'public', but don't auto-enable the onion service.
```

Comment:
What's the rationale here? When would a user want to publish an onion service that doesn't exist?

Reply:
I've extended the comment. We want to set `public = true` as the default for spark-wallet, without enabling the onionService by default.

Reply:
Why's that? One can only publish a hidden service that exists, right?

Reply:
`onionServices` entries that are `public` but not `enabled` are ignored.

Reply:
> We want to set `public = true` as the default for spark-wallet, without enabling the onionService by default.

Can you elaborate why?

Reply:
To quote @jonasnick:
> The "announce" behavior should just be default if the onion service is enabled. 

`public` for spark-wallet has no disadvantages and should be enabled by default when a users sets `onionServices.spark-wallet.enable = true`.

Reply:
Oh, I think I get it. The `enable` is the exposed option the user toggles, which will always include the `.onion` in the QR code through the `true` `public` option.